src/platform/mbed/net/src/BSDSocket.cpp

```diff
@@ -10,33 +10,33 @@ int BSDSocket::open(int family, int type, InternetSocket * socket)
     nsapi_error_t err;
     SocketAddress addr;
 
-    switch (family)
```

Comment:
It is okay for now but I think we should take care of family type in our implementation. I can check this issue for the Cypress board after the merge